doc/contrib/index.rst

```diff
@@ -73,7 +73,16 @@ request.
 Write code
 ~~~~~~~~~~
 
-See :ref:`contribpr`.
+Some tips when contributing code:
+
+-  Consider the :ref:`contribcodingstyle`.
+-  If you add functionality, please update the documentation
+   accordingly.
+-  If you add functionality, add tests if applicable. This helps make
+   sure Arbor is stable and functionality does what it’s supposed to
+   do.
+-  If you add/change the public C++ API, provide Python wrappings.
+-  Make sure Arbor compiles and has no new warnings.
```

Comment:
I would add a point here saying something similar to:
- Add your contribution to the change log (file name goes here)

Reply:
This we don't do (yet), so I'll hold off on this now.